src/utils.js

```diff
@@ -17,6 +17,12 @@
 
 import {ActivityRequestDef} from './activity-types';
 
+/** Only allows http/https URLs. */
+const HTTP_S_ONLY_RE = /^https?\:/i;
+
+/** Disallow obvious unsafe (script) URLs. */
+const OBVIOUS_UNSAFE_URL_RE = /^[^:]*script[^:]*:/i;
```

Comment:
I think this doesn't work, because schemas like this are supported in browsers: "j a v a s c r i p t:"

I would be better to parse the URL instead and look at the schema explicitly.

Reply:
Done. This is otherwise a very preliminary check and additional requirements imposed later on.